package NCbbFast;

message TCbbIpResponceHeader {
    required fixed64 LastCreatedTs = 1;
    required uint32 NumIps = 2;
}

message TCbbIpResponceItem {
    required string Ip = 1;
    required fixed32 ExpiredTs = 2;
}
